test/e2e/test-ollama-auth-proxy-e2e.sh

```
@@ -436,6 +436,81 @@ else
+# Simulate a re-onboard that writes a NEW token to the file but
+# leaves the proxy running with the OLD token.
+ORIGINAL_TOKEN=$(cat "$TOKEN_FILE" 2>/dev/null || echo "")
+DIVERGENT_TOKEN="divergent-$(date +%s)-$(head -c 16 /dev/urandom | xxd -p)"
+
+if [ -n "$ORIGINAL_TOKEN" ]; then
+  info "Original token: ${ORIGINAL_TOKEN:0:16}..."
+  info "Writing divergent token to file: ${DIVERGENT_TOKEN:0:16}..."
+  echo "$DIVERGENT_TOKEN" > "$TOKEN_FILE"
+
+  # Verify proxy still runs with OLD token (divergence exists)
+  OLD_TOKEN_RESULT=$(curl -sf --max-time 3 \
```

Shellcheck (0.11.0)

[warning] OLD_TOKEN_RESULT appears unused. Verify use (or export if used externally).

(SC2034)